src/ml/neural_net/tf_compute_context.cpp

```diff
@@ -201,18 +201,28 @@ class tf_image_augmenter : public float_array_image_augmenter {
 
 tf_image_augmenter::tf_image_augmenter(const options& opts) : float_array_image_augmenter(opts) {}
 
-float_array_image_augmenter::float_array_result tf_image_augmenter::prepare_augmented_images(
+float_array_image_augmenter::float_array_result
+tf_image_augmenter::prepare_augmented_images(
     float_array_image_augmenter::labeled_float_image data_to_augment) {
+  opts_ = tf_image_augmenter::get_options();
```

Comment:
Why is this needed?

Reply:
I am calling this to get options from its base class.